witness/src/Dscp/Witness/CLI.hs

```diff
@@ -70,6 +71,27 @@ netServParamsParser :: Parser NetServParams
 netServParamsParser =
     NetServParams <$> peersParser <*> ourZTNodeIdParser <*> optional internalZTNodeIdParser
 
+----------------------------------------------------------------------------
+-- Metrics server
+----------------------------------------------------------------------------
+
+metricsServerParser :: Parser (Text, Int)
+metricsServerParser =
+  option (maybeReader parseMetricsServer)
+         (long "metrics-server" <>
+          metavar "HOST:PORT" <>
+          value ("127.0.0.1", 8125) <>
```

Comment:
Why is this a default value?

Reply:
What else should be the default?

Reply:
There should be no default, the parameter must be optional.